CommonTools/PileupAlgos/plugins/PuppiPhoton.h

```diff
@@ -37,11 +37,12 @@ class PuppiPhoton : public edm::stream::EDProducer<> {
 	edm::EDGetTokenT< edm::ValueMap<float> > tokenWeights_; 
 	edm::EDGetTokenT< edm::ValueMap<bool>  > tokenPhotonId_; 
 	double pt_;
+	double eta_;
 	bool   usePFRef_;
 	bool   usePhotonId_;
 	std::vector<double>  dRMatch_;
 	std::vector<int32_t> pdgIds_; 
-        std::unique_ptr< PFOutputCollection > corrCandidates_;
+        std::auto_ptr< PFOutputCollection > corrCandidates_;
```

Comment:
@violatingcp : My guess would be that you did not start with the latest 81X version of these files. Make sure you start with the correct file versions before applying your changes.